examples/siro_sandbox/sandbox_app.py

```diff
@@ -625,6 +632,11 @@ def _parse_debug_third_person(args, framebuffer_size):
         action="store_true",
         help="If specified, we use a hybrid social nav design that consists of point nav and social nav. Otherwise, we only use social nav policy purely.",
     )
+    parser.add_argument(
+        "--oracle-follow-human",
+        action="store_true",
+        help="If specified, we use the oracle to follow human.",
```

Comment:
Maybe `If true, use an oracle pointnav policy to follow the human. If false, use a learned social-nav policy.`?

Reply:
Done